ios/Classes/BetterPlayer.m

```diff
@@ -540,15 +538,27 @@ - (void)seekTo:(int)location {
     ///When player is playing, pause video, seek to new position and start again. This will prevent issues with seekbar jumps.
     bool wasPlaying = _isPlaying;
     if (wasPlaying){
-        [_player pause];
+        if (self._willStartPictureInPicture) {
+            // Prevent player pause if has pip mode
+            _player.rate = 0.1;
```

Comment:
> I think, rate is playback speed.

Yes, It's correct. 
I change the rate to 0.1 and the make same behavior as `pause` (in fact when play: rate = 1, stop: rate = 0)

The purpose of this change is to make player that doesn't pause but play at very slow speed (like pause)

Reply:
@mvn-thanhluu-hn 
Thanks. I understand.

Could you add more comments to the code? 🙏 
(Because the code reason is difficult to understand...)

@HiroshiOshiro 
（認識済みかもしれないですが、念のため）
iOS側では、シーク処理実行中も、プレイヤーを再生状態にする（そうしないと、PIPが起動しない）ようです。
コードの意図が分かりづらい部分なので、もうちょっとコメント書いてもらうように、上記で依頼しています。